__tests__/RequestVerifier.spec.ts

```diff
@@ -0,0 +1,95 @@
+import { describe, expect, test, beforeAll } from "@jest/globals";
+import * as moment from 'moment';
+import { RequestVerifier } from '../utils/RequestVerifier'
+
+describe("Request Verifier Class Tests", () => {
```

Comment:
See comment on `SignatureInformation` and apply test structure updates here as well